chapters/ch46.md

```diff
@@ -1,51 +1,52 @@
 > # 46. Reinforcement learning example
 
---> _replace THIS LINE by your translation for the above line_
+# 46. Ví dụ về Học tăng cường
 
 ![img](../imgs/C46_01.png)
 
 > Suppose you are using machine learning to teach a helicopter to fly complex maneuvers. Here is a time-lapse photo of a computer-controller helicopter executing a landing with the engine turned off.
 
---> _replace THIS LINE by your translation for the above line_
+Giả sử như bạn đang sử dụng học máy để dạy trực thăng bay theo những chuyển động phức tạp. Đây là một tấm ảnh time-lapse của một chiếc trực thăng được máy tính điều khiển thực hiện việc hạ cánh khi động cơ đã tắt.
 
 > This is called an "autorotation" maneuver. It allows helicopters to land even if their engine unexpectedly fails. Human pilots practice this maneuver as part of their training. Your goal is to use a learning algorithm to fly the helicopter through a trajectory ​*T* t​hat ends in a safe landing.
 
---> _replace THIS LINE by your translation for the above line_
+Đây được gọi là kĩ thuật "quay tự động". Nó cho phép trực thăng hạ cánh ngay cả khi động cơ bị hỏng ngoài dự kiến. Phi công thực hành kĩ thuật bay này như một phần trong công tác huấn luyện bay. Nhiệm vụ của bạn là sử dụng một thuật toán học tập để lái chiếc trực thăng qua một quỹ đạo *T* và kết thúc với một pha hạ cánh an toàn.
```

Comment:
Hmm, "quay tự động" hay "bay tự động nhỉ"?

Reply:
Mình dịch theo từ điển lạc việt thì bên toán, lý, kỹ thuật nó là "sự quay tự động".
THeo wiki thì đây là khi rotor quay khi không khí di chuyển lên khi trực thăng giảm độ cao làm cánh quạt quay, chứ không phải quay do động cơ. Vậy nên mình nghĩ "quay tự động" là đúng ròi.

Reply:
Ah ok, cảm ơn bạn.